tests/spacier/test_spacier_utils.py

```diff
@@ -94,7 +94,7 @@ def test_get_subjects_of_verb(spacy_doc):
 
 
 def test_get_objects_of_verb(spacy_doc):
-    expected = [[], ["Python"], ["incompatibilities"], [], ["God"]]
+    expected = [[], ["Python"], ["incompatibilities"], ["sake"], ["God", "Overflow"]]
```

Comment:
So, this is what I was wondering about — do these changes make sense, in practice? My grammar is a bit rusty, but I believe "for textacy's sake" is an adverb clause and not an object of the verb "were". And "(Stack) Overflow" is separate from "God" wrt "Thank". I'm not totally sure if this is a bug or feature, expected or unexpected. Could you help me understand this better? 🙏

Reply:
I think I will also have to do some research as to whether this is a better outcome, stay tuned.